data/json/items/tools.json

```diff
@@ -6847,6 +6852,36 @@
     ]
   },
   {
+    "id": "sewing_machine",
```

Comment:
The sewing machine should be a separate PR. It needs to be cleaned up a bit before merging, while the filament change could go in now.
Problems include:
* Doesn't use power
* Isn't any faster than manual sewing
* Uses more materials for same repairs (this one could be an explicit disadvantage, but probably isn't intended)
* Has to be wielded to be used, which is tedious. Should probably wait until it can be used from the floor.